README-dnssec.md

```diff
@@ -0,0 +1,296 @@
+# DNSSEC setup for dnsseed
+
+##### Table of Contents  
+[Requirements](#Requirements)  
+[Software needed](#Software)  
+[Setup](#Setup)  
+[Testing](#Testing)  
+[Example](#Example)  
+[Improvements](#Improvements)  
+[Links](#Links)  
+
+
+<a name="Requirements"/>
+
+## Requirements
```

Comment:
I think it would be useful to note the OS version these instructions were tested on here instead. Like e.g. "Debian 10" instead of having it in the section name below.